props/saber_fett263_buttons.h

```diff
@@ -2390,119 +2626,119 @@ SaberFett263Buttons() : PropBase() {}
-        sound_library_.SayOption();
-        sound_library_.SayNumber(calc_, SAY_WHOLE);
-	break;
-      case RETRACTION_TIME_ARG:
-	menu_type_ = MENU_RETRACTION_TIME;
-	char igt[32];
-	style_parser.GetArgument(current_preset_.GetStyle(blade_num_), RETRACTION_TIME_ARG, igt);
-	ignite_time_ = strtol(igt, NULL, 0);
-	current_preset_.SetStyle(blade_num_,style_parser.SetArgument(current_preset_.GetStyle(blade_num_), RETRACTION_TIME_ARG, "1"));
-	arg_revert_ = strtol(argspace, NULL, 0);
-	break;
-      case RETRACTION_COOL_DOWN_ARG:
-        menu_type_ = MENU_RETRACTION_COOL_DOWN_OPTION;
-        char igc[32];
-        style_parser.GetArgument(current_preset_.GetStyle(blade_num_), RETRACTION_TIME_ARG, igc);
-        ignite_time_ = strtol(igc, NULL, 0);
-        current_preset_.SetStyle(blade_num_,style_parser.SetArgument(current_preset_.GetStyle(blade_num_), RETRACTION_TIME_ARG, "1"));
-        arg_revert_ = strtol(argspace, NULL, 0);
-        sound_library_.SayOption();
-        sound_library_.SayNumber(calc_, SAY_WHOLE);
-        break;
-      case RETRACTION_DELAY_ARG:
-        menu_type_ = MENU_RETRACTION_DELAY;
-        char igd[32];
-        style_parser.GetArgument(current_preset_.GetStyle(blade_num_), RETRACTION_TIME_ARG, igd);
-        ignite_time_ = strtol(igd, NULL, 0);
-        current_preset_.SetStyle(blade_num_,style_parser.SetArgument(current_preset_.GetStyle(blade_num_), RETRACTION_TIME_ARG, "1"));
-        arg_revert_ = strtol(argspace, NULL, 0);
-        break;
-      case LOCKUP_POSITION_ARG:
-	menu_type_ = MENU_LOCKUP_POSITION;
-	ShowColorStyle::SetColor(GetColorArg(blade_num_, LOCKUP_COLOR_ARG));
-	show_lockup_position_.Start(blade_num_);
-	break;
-      case DRAG_SIZE_ARG:
-	menu_type_ = MENU_DRAG_SIZE;
-	ShowColorStyle::SetColor(GetColorArg(blade_num_, DRAG_COLOR_ARG));
-	show_drag_size_.Start(blade_num_);
-	break;
-      case MELT_SIZE_ARG:
-        menu_type_ = MENU_MELT_SIZE;
-        ShowColorStyle::SetColor(GetColorArg(blade_num_, DRAG_COLOR_ARG));
-        show_drag_size_.Start(blade_num_);
-        break;
-      case EMITTER_SIZE_ARG:
-	menu_type_ = MENU_EMITTER_SIZE;
-	ShowColorStyle::SetColor(GetColorArg(blade_num_, EMITTER_COLOR_ARG));
-	show_emitter_size_.Start(blade_num_);
-	break;
-      case PREON_OPTION_ARG:
-	if (!SFX_preon) {
-	  sound_library_.SaySelectOption();
-	} else {
-	  sound_library_.SaySelect();
-	  menu_type_ = MENU_PREON_OPTION;
-	  arg_revert_ = strtol (argspace, NULL, 0);
+        case STYLE_OPTION_ARG:
+          menu_type_ = MENU_STYLE_OPTION;
+          arg_revert_ = strtol (argspace, NULL, 0);
           sound_library_.SayOption();
           sound_library_.SayNumber(calc_, SAY_WHOLE);
-	}
-	break;
-      case PREON_SIZE_ARG:
-	if (!SFX_preon) {
-	  sound_library_.SaySelectOption();
-	} else {
-	  sound_library_.SaySelect();
-	  menu_type_ = MENU_PREON_SIZE;
-	  arg_revert_ = strtol (argspace, NULL, 0);
-	}
-	break;
-      default:
-	break;
+          break;
+        case IGNITION_OPTION_ARG:
```

Comment:
Is this new? It looks new in this diff?
Is it something unrelated? Or is it just moved around?

Reply:
nothing new just fixed indent for the sound_library_ above